packages/adminrouter/extra/src/includes/server/master.conf

```diff
@@ -264,13 +264,26 @@ location ~ ^/(slave|agent)/(?<agentid>[0-9a-zA-Z-]+)(?<url>.+)$ {
 
     more_clear_input_headers Accept-Encoding;
     include includes/proxy-headers.conf;
+
     # Non-streaming endpoints don't require HTTP/1.1 but will work as
     # expected with it enabled. Streaming endpoints require keepalive
```

Comment:
Streaming requires chunked encoding, not keep-alive. Make this comment more accurate